numpy/fft/tests/test_pocketfft.py

```diff
@@ -138,6 +138,15 @@ def test_ihttf(self):
             x_herm, np.fft.ihfft(np.fft.hfft(x_herm, norm="ortho"),
                                  norm="ortho"))
 
+    def test_axes(self):
+        x = random((30, 20, 10))
+        axes = [(0, 1, 2), (0, 2, 1), (1, 0, 2), (1, 2, 0), (2, 0, 1), (2, 1, 0)]
+        for op in [np.fft.fftn, np.fft.ifftn, np.fft.rfftn, np.fft.irfftn]:
```

Comment:
I think parameterizing just this loop might be useful, depending on how likely it is for only some of these to be broken.